### PR TITLE
Update the README.md file to describe the repo

### PR DESCRIPTION
**Pull request by anthonyqzhu**
First commit following the github setup tutorial!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Google Student Training in Engineering Program
 
 This repo contains Anthony Zhu's portfolio and STEP projects.
+
```
Comment (sidharthp) [inline README.md:4]: Did you mean to add an empty line?

Comment (anthonyqzhu) [reply README.md:4]: Yes, the first commit I made I accidentally did it without making a pull request so I wanted to practice making a PR officially

Comment (sidharthp) [reply README.md:4]: Ack.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,17 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Anthony Zhu's Portfolio</h1>
+      <p>Hello! My name is Anthony and I am a rising junior at the University
+          of Michigan. I am studying Computer Science with minors in Math and
+          Philosophy. In my free time I like to play video games, watch movies,
+          play sports, and eat good food.
+      </p>
+      <p>This is my portfolio, where I will show off some of my accomplishments and interests</p>
```
Comment (sidharthp) [inline portfolio/src/main/webapp/index.html]: interests.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -16,6 +16,7 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
```
Comment (sidharthp) [inline portfolio/src/main/webapp/script.js]: You can probably remove this method since its no longer being used?

**Review by sidharthp — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (anthonyqzhu) [reply portfolio/src/main/webapp/script.js]: Ok, I removed the method in latest push

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (anthonyqzhu) [reply portfolio/src/main/webapp/index.html]: Fixed

**Comment by yjennywang** [comment]
Removing myself from the PR to let Sidharth make the final call, but it looks good to me (as a bystander).

**Review by sidharthp — APPROVED** [review]

**Pull request merged**